
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://biolink.github.io/information-resource-ingest-guide/panther_rig/">
      
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.18">
    
    
      
        <title>Panther Gene Orthology and Annotation RIG - Biolink Reference Ingest Guide Schema Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e37652d.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-J23SCD6JB8"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-J23SCD6JB8",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-J23SCD6JB8",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="black" data-md-color-primary="black" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#panther-gene-orthology-and-annotation-rig" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Biolink Reference Ingest Guide Schema Documentation" class="md-header__button md-logo" aria-label="Biolink Reference Ingest Guide Schema Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Biolink Reference Ingest Guide Schema Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Panther Gene Orthology and Annotation RIG
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/biolink/information-resource-ingest-guide" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Biolink Reference Ingest Guide Schema Documentation" class="md-nav__button md-logo" aria-label="Biolink Reference Ingest Guide Schema Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    Biolink Reference Ingest Guide Schema Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/biolink/information-resource-ingest-guide" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../example-rigs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Writing a RIG
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../rig_index/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Index of Existing RIGs
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#source-information" class="md-nav__link">
    <span class="md-ellipsis">
      Source Information
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ingest-information" class="md-nav__link">
    <span class="md-ellipsis">
      Ingest Information
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Ingest Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#relevant-files" class="md-nav__link">
    <span class="md-ellipsis">
      Relevant Files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#included-content" class="md-nav__link">
    <span class="md-ellipsis">
      Included Content
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#filtered-content" class="md-nav__link">
    <span class="md-ellipsis">
      Filtered Content
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#future-content-considerations" class="md-nav__link">
    <span class="md-ellipsis">
      Future Content Considerations
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#target-information" class="md-nav__link">
    <span class="md-ellipsis">
      Target Information
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Target Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#edge-types" class="md-nav__link">
    <span class="md-ellipsis">
      Edge Types
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#node-types" class="md-nav__link">
    <span class="md-ellipsis">
      Node Types
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#future-modeling-considerations" class="md-nav__link">
    <span class="md-ellipsis">
      Future Modeling Considerations
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#provenance-information" class="md-nav__link">
    <span class="md-ellipsis">
      Provenance Information
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="panther-gene-orthology-and-annotation-rig">Panther Gene Orthology and Annotation RIG</h1>
<h2 id="source-information">Source Information</h2>
<p><strong>InfoRes ID:</strong> infores:panther</p>
<p><strong>Description:</strong> The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System  was designed to classify proteins (and their genes) in order to facilitate high-throughput analysis. The core of PANTHER is a comprehensive, annotated library of gene family phylogenetic trees. All nodes in the tree have persistent identifiers that are maintained between versions of PANTHER, providing a stable substrate for annotations of protein properties like subfamily and function.</p>
<p><strong>Citations:</strong>
- https://onlinelibrary.wiley.com/doi/10.1002/pro.4218</p>
<p><strong>Data Access Locations:</strong>
- http://data.pantherdb.org/ftp/</p>
<p><strong>Data Provision Mechanisms:</strong> file_download</p>
<p><strong>Data Formats:</strong> csv</p>
<p><strong>Data Versioning and Releases:</strong> Versioning by number. See https://www.pantherdb.org/data/</p>
<p><strong>Additional Notes:</strong> None</p>
<h2 id="ingest-information">Ingest Information</h2>
<p><strong>Ingest Categories:</strong> primary_knowledge_provider</p>
<p><strong>Utility:</strong> Homology relationships and association of GO and related annotation by orthology inference can be made in between human genes and  non-human species like mouse, rat and many model species, based on the phenotypic characteristics of genes transitively inferred from experimental observations in the model species which cannot generally be easily or ethically replicated upon human beings.</p>
<p><strong>Scope:</strong> Gene to gene genomic orthology relationships and associated annotations</p>
<h3 id="relevant-files">Relevant Files</h3>
<table>
<thead>
<tr>
<th>File Name</th>
<th>Location</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>RefGenomeOrthologs.tar.gz</td>
<td>http://data.pantherdb.org/ftp/ortholog/current_release/</td>
<td>Gene to Gene Orthology Relationships in reference genomes</td>
</tr>
<tr>
<td>PTHR<Panther-version>_<taxon_name></td>
<td>http://data.pantherdb.org/ftp/sequence_classifications/current_release/PANTHER_Sequence_Classification_files/</td>
<td>Gene sequence annotation from specific genomes</td>
</tr>
</tbody>
</table>
<h3 id="included-content">Included Content</h3>
<table>
<thead>
<tr>
<th>File Name</th>
<th>Included Records</th>
<th>Fields Used</th>
</tr>
</thead>
<tbody>
<tr>
<td>RefGenomeOrthologs.tar.gz</td>
<td>All records, with taxonomic filtering noted below.</td>
<td>Gene, Ortholog, Type of ortholog, Panther Ortholog ID</td>
</tr>
<tr>
<td>PTHR<Panther-version>_<taxon_name></td>
<td>All records for the specified taxon.</td>
<td>Gene, Ortholog, Type of ortholog, Panther Ortholog ID</td>
</tr>
</tbody>
</table>
<h3 id="filtered-content">Filtered Content</h3>
<table>
<thead>
<tr>
<th>File Name</th>
<th>Filtered Records</th>
<th>Rationale</th>
</tr>
</thead>
<tbody>
<tr>
<td>RefGenomeOrthologs.tar.gz, PTHR<Panther-version>_<taxon_name></td>
<td>All records with Gene and Ortholog pairwise annotated with taxon name as 'HUMAN', 'MOUSE' or 'RAT' specific.</td>
<td>Panther contains a huge number of records covering orthologs across 144 diverse species (as of September 2025), but our core interest in Translator focuses on genes in taxa close in evolutionary terms to human, thus having significant genetic, molecular and physiological  annotation closer to human biology in character. It is also expected that the genes of these evolutionarily close species also already have a significant assignment of functional roles  partially inferred from other model organisms (e.g. developmental gene functions mapped  from fruit fly or nematode onto mouse genes, but also experimentally tested in mouse). The character of genetic and physiological systems are much more similar between humans and  mice or rats, in particular, studied responses in pharmacology, metabolism, and the immune system.</td>
</tr>
</tbody>
</table>
<h3 id="future-content-considerations">Future Content Considerations</h3>
<p><strong>other:</strong> Additional model species may be included in the future.</p>
<h2 id="target-information">Target Information</h2>
<p><strong>Target InfoRes ID:</strong> infores:translator-panther-kgx</p>
<h3 id="edge-types">Edge Types</h3>
<table>
<thead>
<tr>
<th>Subject Categories</th>
<th>Predicate</th>
<th>Object Categories</th>
<th>Knowledge Level</th>
<th>Agent Type</th>
<th>UI Explanation</th>
</tr>
</thead>
<tbody>
<tr>
<td>biolink:Gene</td>
<td>biolink:orthologous_to</td>
<td>biolink:Gene</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther orthology family</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:has_part</td>
<td>biolink:GeneFamily</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther gene family to gene family relationship</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:has_part</td>
<td>biolink:Gene</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther Gene family membership</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:located_in</td>
<td>biolink:CellularComponent</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther Gene family cellular location of expression</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:actively_involved_in</td>
<td>biolink:BiologicalProcess</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther Gene Family involvement in a biological process</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:catalyzes</td>
<td>biolink:MolecularActivity</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther Gene Family catalysis of a molecular activity</td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>biolink:actively_involved_in</td>
<td>biolink:Pathway</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther active involvement of a Gene Family in a pathway</td>
</tr>
<tr>
<td>biolink:Pathway</td>
<td>biolink:has_participant</td>
<td>biolink:GeneFamily</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther participation of a Gene Family in a pathway</td>
</tr>
<tr>
<td>biolink:Pathway</td>
<td>biolink:subclass_of</td>
<td>biolink:BiologicalProcess</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther pathway as a subclass of a biological process</td>
</tr>
<tr>
<td>biolink:Pathway</td>
<td>biolink:subclass_of</td>
<td>biolink:Pathway</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther pathway as a subclass of another pathway</td>
</tr>
<tr>
<td>biolink:MolecularActivity</td>
<td>biolink:subclass_of</td>
<td>biolink:MolecularActivity</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther molecular activity as a subclass of another molecular activity</td>
</tr>
<tr>
<td>biolink:BiologicalProcess</td>
<td>biolink:subclass_of</td>
<td>biolink:BiologicalProcess</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther biological process as a subclass of another biological process</td>
</tr>
<tr>
<td>biolink:CellularComponent</td>
<td>biolink:subclass_of</td>
<td>biolink:CellularComponent</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther cellular component as a subclass of another cellular component</td>
</tr>
<tr>
<td>biolink:CellularComponent</td>
<td>biolink:subclass_of</td>
<td>biolink:AnatomicalEntity</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther cellular component as a subclass of anatomical entity</td>
</tr>
<tr>
<td>biolink:CellularComponent</td>
<td>biolink:subclass_of</td>
<td>biolink:GrossAnatomicalStructure</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther cellular component as a subclass of gross anatomical structure</td>
</tr>
<tr>
<td>biolink:AnatomicalEntity</td>
<td>biolink:subclass_of</td>
<td>biolink:CellularComponent</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther anatomical entity as a subclass of cellular component</td>
</tr>
<tr>
<td>biolink:GrossAnatomicalStructure</td>
<td>biolink:subclass_of</td>
<td>biolink:CellularComponent</td>
<td>knowledge_assertion</td>
<td>manual_validation_of_automated_agent</td>
<td>Panther gross anatomical structure as a subclass of cellular component</td>
</tr>
</tbody>
</table>
<h3 id="node-types">Node Types</h3>
<table>
<thead>
<tr>
<th>Node Category</th>
<th>Source Identifier Types</th>
<th>Additional Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>biolink:Gene</td>
<td>HGNC, MGI, RGD, ENSEMBL, NCBIGene</td>
<td></td>
</tr>
<tr>
<td>biolink:GeneFamily</td>
<td>PANTHER.FAMILY</td>
<td></td>
</tr>
<tr>
<td>biolink:Pathway</td>
<td>PANTHER.PATHWAY</td>
<td></td>
</tr>
<tr>
<td>biolink:MolecularActivity</td>
<td>GO</td>
<td></td>
</tr>
<tr>
<td>biolink:BiologicalProcess</td>
<td>GO</td>
<td></td>
</tr>
<tr>
<td>biolink:CellularComponent</td>
<td>GO</td>
<td></td>
</tr>
<tr>
<td>biolink:AnatomicalEntity</td>
<td>UBERON, UMLS</td>
<td></td>
</tr>
<tr>
<td>biolink:GrossAnatomicalStructure</td>
<td>UBERON</td>
<td>biolink:Attribute is used to capture the information content</td>
</tr>
</tbody>
</table>
<h3 id="future-modeling-considerations">Future Modeling Considerations</h3>
<p><strong>other:</strong> The Monarch Initiative ingest of Panther data (https://github.com/monarch-initiative/pantherdb-orthologs-ingest) sometimes attempts to map eccentric identifiers to the NCBI gene identifier space. The initial iteration of  the Panther ingest in Translator does not attempt to do this at this time, but rather simply uses the given UniProt identifier.</p>
<h2 id="provenance-information">Provenance Information</h2>
<p><strong>Contributors:</strong>
- Richard Bruskiewich - data modelling, domain expertise, code author
- Kevin Schaper - Phase 2 legacy code expert
- Evan Morris - Phase 2 legacy code expert
- Chunlei Wu - Phase 2 legacy code expert
- Matt Brush - data modelling</p>
<p><strong>Artifacts:</strong>
- Ingest Survey (https://docs.google.com/spreadsheets/d/1YlpI5bjGNGR5JC9VWxZJ7dd87hS_b4BMZv5geYe2NCk/edit?gid=0#gid=0)
- Ingest Ticket (https://github.com/NCATSTranslator/Data-Ingest-Coordination-Working-Group/issues/44)</p>









  




                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.92b07e13.min.js"></script>
      
    
  </body>
</html>